msos/WaitChains.cs

```diff
@@ -56,8 +56,7 @@ public void Execute(CommandExecutionContext context)
 
         private void SetStrategy()
         {
-            if (_context.TargetType == TargetType.DumpFile
-                || _context.TargetType == TargetType.DumpFile)
```

Comment:
I see.
I ran some tests with (MiniDump file) which is set `_context.TargetType `to `DumpFileNoHeap `- I got this output:
> This command is not supported for the current target type: 'DumpFileNoHeap' 
I think that there is  a missing parameter in WaitChain.cs attrubute:
`[SupportedTargets(TargetType.DumpFile, TargetType.LiveProcess)]`
should be changed to:
`[SupportedTargets(TargetType.DumpFile, TargetType.LiveProcess, TargetType.DumpFileNoHeap)]`

Tested, seems to work :)